### DISCUSSION TITLE
GITHUB_WORKSPACE not mounted in Actions Docker container

### DISCUSSION THREAD

**Discussion by paambaati** [community/community#26125] [Actions]
<p>I’m trying to develop my first GitHub Action that uploads code coverage for Node.js projects to Code Climate.</p>
<p>I have a draft version at <a href="https://github.com/paambaati/codeclimate-action" rel="nofollow noopener">https://github.com/paambaati/codeclimate-action</a>, and have it set up as an action on another project at <a href="https://github.com/paambaati/websight/runs/189324965" rel="nofollow noopener">https://github.com/paambaati/websight</a></p>
<p>Now when I run the action (see one such run at <a href="https://github.com/paambaati/websight/runs/189324965" rel="nofollow noopener">https://github.com/paambaati/websight/runs/189324965</a>), the <code>GITHUB_WORKSPACE</code> doesn’t seem to be mounted like the documentation says. You can see that <code>/github</code> doesn’t exist, and <code>env</code> doesn’t seem to print any <code>GITHUB_*</code> variables either.</p>
<p>What am I doing wrong?</p>

**Comment by paambaati** [comment, marked as answer]
<p><strong>Resolved.</strong></p>
<p>I was looking at the wrong place for the variables. They’re available inside the entrypoint and not the Dockerfile itself.</p>

**Comment by weironz** [comment]
it's not support, you can use https://github.com/marketplace/actions/docker-run-action
